### PR TITLE
Added isSocial parameter in creating an instance to allow social sharing

### PR DESCRIPTION
**Pull request by singhsanket143**
The pull request is implementation for the feature listed in issue #16368. During the instance creation, there is an option added in the advanced options to enable or disable social sharing of an instance.

image

### CODE REVIEW

File: client/views/create/create.html [hunk 1]
```diff
@@ -74,6 +74,13 @@
 	                    <option value="hidden">Hidden</option>
```
Comment (peter-hank) [inline client/views/create/create.html]: @singhsanket143 can you fix the code formatting of this file (indentation mostly)? It looks horrible ATM.

File: client/views/create/create.js [hunk 2]
```diff
@@ -90,6 +90,9 @@ Template.create.events({
     const maxResponse = responseSelect[responseSelect.selectedIndex].value;
     const hiddenSelector = document.getElementsByName('visibility')[0];
     const isHidden = (hiddenSelector[hiddenSelector.selectedIndex].value === 'hidden');
+    const socialSelector = document.getElementsByName("social")[0]; // Extract the HTML select element for getting the social toggles
```
Comment (peter-hank) [inline client/views/create/create.js]: @singhsanket143 can you use the single quote as in the `hiddenSelector` var?

File: client/views/create/create.js [hunk 3]
```diff
@@ -90,6 +90,9 @@ Template.create.events({
...
+    const isSocial = socialSelector[socialSelector.selectedIndex].value === "on"; // Boolean value to store whether the user has selected Social share to be on or off
```
Comment (peter-hank) [inline client/views/create/create.js]: The same here, use `'on'`, just to be consistent with the rest of the code in the file.

**Review by peter-hank — CHANGES_REQUESTED** [review]

File: client/views/create/create.js (hunk 3 above)
Comment (singhsanket143) [reply client/views/create/create.js]: @peter-hank Updated the quotes here.

File: client/views/create/create.html (hunk 1 above)
Comment (singhsanket143) [reply client/views/create/create.html]: Updated the formatting for this file.

File: client/views/create/create.js (hunk 2 above)
Comment (singhsanket143) [reply client/views/create/create.js]: Updated the quotes here.

File: client/views/create/create.html (hunk 1 above)
Comment (peter-hank) [reply client/views/create/create.html]: @singhsanket143 can you fix it in the whole file?

Comment (singhsanket143) [reply client/views/create/create.html]: @peter-hank I have updated the whole file. Please review it

File: client/views/create/create.js [hunk 4]
```diff
@@ -90,6 +90,9 @@ Template.create.events({
     const maxResponse = responseSelect[responseSelect.selectedIndex].value;
     const hiddenSelector = document.getElementsByName('visibility')[0];
     const isHidden = (hiddenSelector[hiddenSelector.selectedIndex].value === 'hidden');
+    const socialSelector = document.getElementsByName('social')[0]; // Extract the HTML select element for getting the social toggles
```
Comment (peter-hank) [inline client/views/create/create.js]: @singhsanket143 can you move the two comments up?

Comment (singhsanket143) [reply client/views/create/create.js]: @peter-hank Updated the comments

**Pull request merged**